src/mlx90632.c

```diff
@@ -449,4 +471,43 @@ int32_t mlx90632_init(void)
 
     return 0;
 }
+
+int32_t mlx90632_start_measurement_burst(void)
+{
+    int32_t ret;
+    int tries = 150;
+    uint16_t reg;
+
+    ret = mlx90632_i2c_read(MLX90632_REG_CTRL, &reg);
```

Comment:
There is a race condition here. Can we think of the other way?

Reply:
Also keep in mind to do a cleanup in case of error (we want old value back right?) Or make a switch to step mode a separate function, so that people can call it to switch mode and just use other functions to get data...

Reply:
Well, we need to set the SOB bit to start a new measurement in sleeping step mode. I am not sure I see another way. I think there will be no race condition since this function is called only in sleeping step mode and the CTRL_REG should have 'stable' values. I think that if the write to the register fails, the old value is still in the register, so we do not need to re-write it.